tests/load-tests/ci-scripts/load-test.sh

```diff
@@ -5,20 +5,20 @@ set -o errexit
 set -o pipefail
 
 # shellcheck disable=SC1090
-source "/usr/local/ci-secrets/redhat-appstudio-load-test/load-test-scenario.${1:-concurrent}"
+source "/usr/local/ci-secrets/konflux-ci-load-test/load-test-scenario.${1:-concurrent}"
```

Comment:
This (and all other places that take credentials from our vault) should corellate with https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-appstudio/load-test/redhat-appstudio-load-test-ref.yaml#L16

Reply:
I did that because I got this error:
```
./tests/load-tests/ci-scripts/setup-cluster.sh: line 8: /usr/local/ci-secrets/konflux-ci-load-test/load-test-scenario.jhutar: No such file or directory 
```
Now with that `find` it was probably some temporary issue.